shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/FeaturePromo.kt

```diff
@@ -0,0 +1,17 @@
+package com.mbta.tid.mbta_app.model
+
+enum class FeaturePromo(val addedInVersion: AppVersion) {
+    // ktfmt does not like empty enums as of the version we get with spotless
+    Nothing(AppVersion(0u, 0u, 0u));
```

Comment:
suggestion(non-blocking): To confirm my understanding, having a single AppVersion works for a feature promo b/c we have previously discussed that iOS and android should only ever differ in the patch number, correct? With that, small suggestion to document that in the deploy section of the readme.

Reply:
That combined with the principle that any feature worth promoting is also worth bumping the minor version number for, yeah. That said, though, promoting combined stop + trip details on Android would be incorrect, since it never had anything else, and representing that accurately under the assumption that version numbers are only ever synchronized would be somewhat difficult, so it may be simpler to just allow separate version numbers to be used.